Smooch.postman_collection.json

```
@@ -6289,6 +6189,84 @@
 									"body": "{\n    \"integration\": {\n        \"_id\": \"5735ddfd48011972d621dc0a\",\n        \"type\": \"wechat\",\n        \"appId\": \"c69175d6d125b772b\",\n        \"webhookSecret\": \"3889794ab2fd4a70940a97c4b4a6372e\"\n    }\n}"
 								}
 							]
+						},
+						{
+							"name": "WhatsApp",
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Authorization",
+										"value": "Bearer {{JWT}}"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n\t\"type\": \"whatsapp\",\n    \"deploymentId\": \"<whatsappDeploymentId>\"\n}"
+								},
+								"url": {
+									"raw": "{{url}}/{{apiVersion}}/apps/{{appId}}/integrations",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"{{apiVersion}}",
+										"apps",
+										"{{appId}}",
+										"integrations"
+									]
+								},
+								"description": "### For more information\n- Guide Docs: [https://docs.smooch.io/guide/whatsapp/#how-to-enable-the-channel](https://docs.smooch.io/guide/whatsapp/#how-to-enable-the-channel)\n- API Docs: [https://docs.smooch.io/rest/#integration-schemas-whatsapp](https://docs.smooch.io/rest/#integration-schemas-whatsapp)"
+							},
+							"response": [
+								{
+									"name": "WhatsApp",
+									"originalRequest": {
+										"method": "POST",
+										"header": [
+											{
+												"key": "Authorization",
+												"value": "Bearer {{JWT}}"
+											},
+											{
+												"key": "Content-Type",
+												"name": "Content-Type",
+												"value": "application/json",
+												"type": "text"
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": "{\n\t\"type\": \"whatsapp\",\n    \"deploymentId\": \"<whatsappDeploymentId>\"\n}",
```

Une p'tite dernière:

```suggestion
											"raw": "{\n    \"type\": \"whatsapp\",\n    \"deploymentId\": \"<whatsappDeploymentId>\"\n}",
```